crates/flox/src/commands/environment.rs

```diff
@@ -450,11 +472,11 @@ impl Install {
                 if let Some(false) = installation.already_installed.get(pkg) {
                     info!("✅ '{pkg}' installed to environment {description}");
                 } else {
-                    info!("🛑 '{pkg}' already installed to environment {description}");
+                    info!("⚠️ '{pkg}' already installed to environment {description}");
```

Comment:
@zmitchell I thought 🛑 felt a bit more like there was an error, but can revert if you prefer